tests/foreman/api/test_operatingsystem.py

```diff
@@ -110,11 +111,14 @@ def test_positive_create_with_os_family(self):
 
         :expectedresults: Operating system entity is created and has proper OS
             family assigned
-
+        :CaseAutomation: Automated
```

Comment:
@vikask18 , You forgot to add blank line before and after this line, please do that.

Reply:
Done.